frontend/src/components/pages/tasks/TaskModal.tsx

```diff
@@ -231,7 +244,7 @@ const TaskModal = ({ isOpen, setIsOpen }: Props): React.ReactElement => {
           onBlur={handleMoneyInput}
           submitPressed={submitPressed}
           leftElement="$"
-          required
+        
```

Comment:
Add in a "+ Create Comment" option if possible!
<img width="139" alt="Screenshot 2024-10-24 at 7 56 33 PM" src="https://github.com/user-attachments/assets/e18dc4c5-01b2-42ad-9393-263096aa21c3">